tox.ini

```diff
@@ -24,9 +28,7 @@ commands =
 [testenv:test]
 allowlist_externals = coverage
 commands =
-    coverage run --omit='tests/*' -m pytest -vv tests/
-    coverage xml -i
-    coverage report
+    python -m pytest --cov-report=xml --cov-report=term --cov -n auto -vv tests/
```

Comment:
I cannot find on the web what is this `auto` command of `pytest`.
Can you describe where you read about it @KAUTH?

Reply:
https://pytest-xdist.readthedocs.io/en/stable/distribution.html#running-tests-across-multiple-cpus

Reply:
It's a `pytest` pluging called `pytest-xdist`, as Kairo shared above, that allows you to distribute your `pytest` tests across multiple CPUs.